openmetadata-docs/content/v1.7.x-SNAPSHOT/getting-started/day-1/hybrid-saas/hybrid-ingestion-runner.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Hybrid Ingestion Runner
+slug: /getting-started/day-1/hybrid-saas/hybrid-ingestion-runner.md
+collate: true
+---
+
+# Hybrid Ingestion Runner
+
+The **Hybrid Ingestion Runner** is a component designed to enable Collate customers operating in hybrid environments to securely execute ingestion workflows within their own cloud infrastructure. In this setup, your SaaS instance is hosted on Collate’s cloud, while tools like Argo Workflows are deployed within the your private cloud. The Hybrid Runner acts as a bridge between these two environments, allowing ingestion workflows to be triggered and managed remotely—without requiring the customer to share secrets or sensitive credentials with Collate. It securely receives workflow execution requests and orchestrates them locally, maintaining full control and data privacy within the customer’s environment.
+
+## Prerequisites
+
+Before setting up the Hybrid Ingestion Runner, ensure the following:
+
+- Your Collate SaaS instance is available.
+- You have **Argo Workflows** deployed in your own cloud environment.
```

Comment:
Is there some command that can be run to test this? Or maybe link to our docs or resource that explain this.